daisy_tutorials/modify_image/README.md

```diff
@@ -0,0 +1,197 @@
+# Modifying a base image using Daisy
+
+Daisy workflows [have many
+capabilities,](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy#workflow-config-overview)
+and it is hard to know where to start. This tutorial will demonstrate
+writing a very simple workflow to modify a GCE base image.
+
+The overall steps of the workflow will be:
+
+1. Create a new instance
+2. Run a provisioning script to modify the instance
+3. Create a new disk image from the instance
+4. Delete the instance
+
+## Provisioning the instance
+
+What you use to provision the instance is open and not dependant on
+Daisy. In this simple case, we are using a simple shell script to
+install Emacs. Because we are creating a CentOS image, we simply call
+`yum install`. The provisioning script is named `emacs_install.sh`.
+
+```shell
+#!/bin/bash
+
+yum -y install emacs-nox
+shutdown -h now
+```
+
+The script shuts down the instance after it is done. We will use this
```

Comment:
This is great as a basic example. I wonder if we should do a more complex example with a few different steps in the script- using serial port output and success/failure cases to show how to fail or succeed on different parts of your script. Obviously- in a more advanced example tutorial.

Reply:
Thanks, yes sounds like a good second tutorial.